src/main/java/io/jenkins/plugins/checks/github/GitHubChecksDetails.java

```
@@ -0,0 +1,91 @@
+            case NEUTRAL: return Conclusion.NEUTRAL;
+            case SUCCESS: return Conclusion.SUCCESS;
+            case ACTION_REQUIRED: return Conclusion.ACTION_REQUIRED;
+            default: return null;
+        }
+    }
+
+    public Output getOutput() {
+        ChecksOutput checksOutput = details.getOutput();
+        Output output = new Output(checksOutput.getTitle(),
+                checksOutput.getSummary());
+        output.withText(checksOutput.getText());
+        checksOutput.getChecksAnnotations().stream().map(this::getAnnotation).forEach(output::add);
+        checksOutput.getChecksImages().stream().map(this::getImage).forEach(output::add);
+        return output;
+    }
+
+    private Annotation getAnnotation(final ChecksAnnotation checksAnnotation) {
+        return new Annotation(checksAnnotation.getPath(),
+                checksAnnotation.getStartLine(), checksAnnotation.getEndLine(),
+                getAnnotationLevel(checksAnnotation.getAnnotationLevel()),
+                checksAnnotation.getMessage())
+                .withTitle(checksAnnotation.getTitle())
+                .withRawDetails(checksAnnotation.getRawDetails())
+                .withStartColumn(checksAnnotation.getStartColumn())
+                .withEndColumn(checksAnnotation.getEndColumn());
+    }
+
+    private Image getImage(final ChecksImage checksImage) {
+        return new Image(checksImage.getAlt(), checksImage.getImageUrl()).withCaption(checksImage.getCaption());
+    }
+
+    private AnnotationLevel getAnnotationLevel(final ChecksAnnotationLevel checksLevel) {
+        // FIXME: should create a lookup table?
+        switch (checksLevel) {
+            case NOTICE:
+                return AnnotationLevel.NOTICE;
+            case FAILURE:
+                return AnnotationLevel.FAILURE;
+            case WARNING:
+                return AnnotationLevel.WARNING;
+            default:
+                return null;
```

probably want to throw an exception here?